rust/noosphere-gateway/src/handlers/v0alpha1/fetch.rs

```diff
@@ -4,14 +4,15 @@ use anyhow::Result;
 
 use axum::{body::StreamBody, extract::Query, http::StatusCode, Extension};
 use bytes::Bytes;
-use noosphere_api::data::FetchParameters;
 use noosphere_core::{
+    api::v0alpha1::FetchParameters,
```

Comment:
similar to the routes, I'd expect the parameters to consistently come from whatever the current API version is e.g. `api::v0alpha2::FetchParameters`

Reply:
See comment above.